Autopilot/AttitudeManager/net.h

```diff
@@ -15,13 +15,13 @@ extern "C" {
 #endif
 
 /** Time in miliseconds for how often a P0(high priority) packet gets sent down. Default=300 **/
-#define P0_SEND_FREQUENCY 200 
+#define P0_SEND_FREQUENCY 300 
```

Comment:
No, just realized it shouldn't have been changed. Didn't realize it was in this PR. Thought it was the last one. Will revert. Serge's PR changes this anyway.